packages/lib/storage/src/new/RemoteInterface.ts

```diff
@@ -0,0 +1,37 @@
+import { stringify } from 'dm3-lib-shared';
+import { Chunk, Encryption, KeyValueStore } from './types';
+import axios from 'axios';
+
+/**
+ * This function creates a key-value store which can be used to store
+ * and retrieve data from a remote server.
+ *
+ * @param {string} baseUrl - The base url of the remote server.
+ * The base URL must include the ENS account name of the user.
+ * @param {Encryption} [encryption] - An encryption object to encrypt and decrypt values.
+ * @returns {KeyValueStore} A key-value store object.
+ */
+export function createRemoteKeyValueStoreApi(
+    baseUrl: string,
+    encryption?: Encryption,
+): KeyValueStore {
+    return {
+        write: async (key: string, value: Chunk) => {
+            const valueToStore = encryption
+                ? await encryption.encrypt(stringify(value))
```

Comment:
I think we could get rid of the if statement by using a default parameter that either is the dm3 encryption or a function that jus returns the input.

class Dm3Encryption implements Encryption {
...
}

export function createRemoteKeyValueStoreApi(
    baseUrl: string,
    encryption: Encryption = dm3Encryption,
): KeyValueStore {
   read: async <T extends Chunk>(key: string) => {
            const result = (await axios.get(`${baseUrl}/${key}`)).data;
            if (!result) {
                return undefined;
            }
            const processedResult =
           await encryption.decrypt(result.toString())

            return JSON.parse(processedResult) as T;
        },
    };